pkg/kritis/review/validating_transport.go

```diff
@@ -52,7 +52,7 @@ func (avt *AttestorValidatingTransport) GetValidatedAttestations(image string, a
 		glog.Error(err)
 		return nil, err
 	}
-	attestations, err := avt.Client.Attestations(image, attestationAuthorities)
+	attestations, err := avt.Client.Attestations(image, aa)
```

Comment:
Please use avt.Attestor here instead of 'aa' (and remove 'aa' from the arguments).

AttestorValidatingTransport already "knows" where to get the attestations from (that is the Note).

Reply:
This has now been addressed